### ISSUE TITLE
Error using GO! Command - SQL Server module installation

### ISSUE THREAD

**Issue by nickrobbo** [NAVBaaS/GitIntegration#1]
Hi, 

I have installed your extension, I get the following error when using the Go command, when the application is trying to install the required Power Shell modules.

```
Started function: Installing module NavContainerHelper and SqlServer from the PowerShell Gallery.
PackageManagement\Get-Package : No match was found for the specified search criteria and module names 'SqlServer'.
At C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1.0.0.1\PSModule.psm1:2253 char:9
+         PackageManagement\Get-Package @PSBoundParameters | Microsoft. ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Microsoft.Power...lets.GetPackage:GetPackage) [Get-Package], Exception
    + FullyQualifiedErrorId : NoMatchFound,Microsoft.PowerShell.PackageManagement.Cmdlets.GetPackage
```

**Comment by richardrobberse** [comment]
I will look into this and ship a new version a.s.a.p.
As a workaround, you can use Install-PackageProvider -Name 'NuGet', but you probably already did that.

**Comment by nickrobbo** [comment]
Yes, I installed the module manually, which resolved the issue when I used the command again.

**Comment by richardrobberse** [comment]
Fixed in version 1.1.0.
NuGet will be installed when using the extension for the first time.
